core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.ts

```diff
@@ -1,23 +1,41 @@
-import { CommonModule } from '@angular/common';
-import { Component, OnInit, inject } from '@angular/core';
+import { CommonModule, DOCUMENT } from '@angular/common';
+import {
+    AfterViewInit,
+    Component,
+    ElementRef,
+    Inject,
+    OnInit,
+    ViewChild,
+    inject
+} from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { ActivatedRoute, Params, Router } from '@angular/router';
 
+import { Dialog } from 'primeng/dialog';
+
+import { DotCMSContentlet } from '@dotcms/dotcms-models';
 import { SafeUrlPipe } from '@dotcms/ui';
 
 import { EditEmaStore } from './store/dot-ema.store';
 
 import { DotPageApiService } from '../services/dot-page-api.service';
+import { CUSTOM_EVENTS, MESSAGE_ACTIONS } from '../shared/models';
 
 @Component({
     selector: 'dot-ema',
     standalone: true,
+
     imports: [CommonModule, FormsModule, SafeUrlPipe],
```

Comment:
Needs to import the `DialogModule` here.

Reply:
Done!